src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/00 About scenarios/02 VU allocation.md

```diff
@@ -0,0 +1,88 @@
+---
+title: VU allocation
+excerpt: How k6 allocates VUs in the open-model, arrival-rate executors
+---
+
+This document explains how k6 allocates VUs in the arrival-rate executors.
+
+In arrival-rate executors, three properties determine the iterations per second:
+-  `rate`  determines how many iterations k6 starts.
+- `timeUnit` determines how frequently it starts the number of iterations.
+- `preAllocatedVUs` sets the number of VUs to use to reach the target iterations per second.
+
+In short, while `rate` and `timeUnit` set the target iterations per second, **you must allocate enough VUs to reach this target.**
+By pre-allocating VUs, k6 can initialize the VUs necessary for the test before the test runs,
+which ensures that the CPU cost of allocation doesn't interfere with test execution.
+
+```javascript
+export const options = {
+  scenarios: {
+    constant_load: {
+      executor: "constant-arrival-rate",
+      preAllocatedVUs: 4,
+      rate: 8,
+      timeUnit: "1m",
+    },
+  },
+};
+```
+
+
+<Blockquote mod="attention" title="">
+
+In cloud tests, **both `preAllocatedVUs` and `maxVUs` count against your subscription.**
```

Comment:
I don't want to move the whole block down, because it's in an admonition and I think it's most respectful if we alert readers about things that can affect subscription use up front. But I'll move your addition to that section.

![image](https://user-images.githubusercontent.com/47385188/212163638-0265b06b-1a8a-49e9-abe6-d6cc9d81bbfb.png)

Reply:
I just realized this might be confusing in another way, "both `preAllocatedVUs` and `maxVUs` count against your subscription" may confuse users that `preAllocatedVUs + maxVUs` will be counted against their subscription, while the reality is that just the bigger number between the two (i.e. `max(preAllocatedVUs, maxVUs)`) will be counted.

And, given that `maxVUs` will always be `>= preAllocatedVUs`, and if you don't specify `maxVUs` explicitly, it's implicitly equal to `maxVUs`, `maxVUs` is what will be counted against the subscription

Reply:
Maybe:

> The number of VUs you allocate (including `maxVUs`) counts against your subscription.